### PR TITLE
Import tkinter

### PR DESCRIPTION
**Pull request by arokem**
Hey @Garyfallidis - maybe this will help with the troubles we were having running the travis tests on #587?

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -41,21 +42,27 @@ before_install:
...
       fi
     - if [ "${VTK}" == "1" ]; then
-      sudo apt-get update;
-      sudo apt-get install -y python-vtk;
-      sudo apt-get install -y xvfb;
-      sudo apt-get install -y python-tk;
-      python -c "import vtk";
+          sudo apt-get update;
+          sudo apt-get install -y python-vtk;
+          sudo apt-get install -y xvfb;
```
Comment (MarcCote) [inline .travis.yml]: Didn't know that `xvfb` existed, nice.

**Comment by Garyfallidis** [comment]
Travis says there is no numpy.

**Comment by arokem** [comment]
Yet numpy is installed: https://travis-ci.org/nipy/dipy/jobs/86104576#L435

Apparently not on the same python though. Ugh.

File: .travis.yml (hunk 1 above)
Comment (Garyfallidis) [inline .travis.yml]: Yeah, cool tool.

**Pull request closed without merging**